### ISSUE TITLE
Meaning of 'tr_g, rot_g, tor_g' and 'tr_perturb, rot_perturb, tor_perturb' in 'sampling.py' ?

### ISSUE THREAD

**Issue by YoungjunLee13** [gcorso/DiffDock#168]
1. I can see the snippet of code below is implementing Algorithm4 in the Appendix B of your paper, but cannot understand which symbols in the paper `tr_g, rot_g, tor_g` are representing. **Q. Could you explain a bit about why you introduced them?**

```
tr_g = tr_sigma * torch.sqrt(torch.tensor(2 * np.log(model_args.tr_sigma_max / model_args.tr_sigma_min)))
rot_g = 2 * rot_sigma * torch.sqrt(torch.tensor(np.log(model_args.rot_sigma_max / model_args.rot_sigma_min)))
# tr_g = 61.55, rot_g = 6.16

if ode:
    tr_perturb = (0.5 * tr_g ** 2 * dt_tr * tr_score.cpu()).cpu()
    rot_perturb = (0.5 * rot_score.cpu() * dt_rot * rot_g ** 2).cpu()
else:
    tr_z = torch.zeros((b, 3)) if no_random or (no_final_step_noise and t_idx == inference_steps - 1) \
        else torch.normal(mean=0, std=1, size=(b, 3))
    # shape = ([2, 3])
    tr_perturb = (tr_g ** 2 * dt_tr * tr_score.cpu() + tr_g * np.sqrt(dt_tr) * tr_z).cpu()
    # shape = ([2, 3])

    rot_z = torch.zeros((b, 3)) if no_random or (no_final_step_noise and t_idx == inference_steps - 1) \
        else torch.normal(mean=0, std=1, size=(b, 3))
    # shape = ([2, 3])
    rot_perturb = (rot_score.cpu() * dt_rot * rot_g ** 2 + rot_g * np.sqrt(dt_rot) * rot_z).cpu()
    # shape = ([2, 3])
```

2. Judging from the fact that you are using SDE as $`dx = \sqrt{d\sigma^2/dt} dw`$, and NCSN(introduced in Generative Modeling by Estimating Gradients of the Data Distribution, Yang Song) is a discretized version of the SDE(as explained in Score-based Gererative Modeling through Stochastic Differential Equations), I suspect there should be a lot in common between training and inference schemes of your paper and Yang Song's paper. Some examples of similarity that led me to this thought are listed below. **Q. So if I understand everything correctly, I wonder why you didn't implement anything related to '$`\lambda\left(\sigma_i\right)`$' of expression (6) of Yang Song's, in your code.**

    1) 'Iteration over $`x^*`$(or $`x_0`$)' in Algorithm3 of your paper   <->   $\mathbb{E_{p_data(x)}}$ of the expression (5) of Yang Song's
    2) 'Sample t ~ Uni([0, 1])' in Algorithm3 of your paper   <->   '$` \frac{1}{L} \Sigma`$' of the expression (6) of Yang Song's
    3) 'Sample $`\Delta r, \Delta R, \Delta \theta`$ from diffusion kernels $`p_{t}^{tr} \left(\cdot|0\right), p_{t}^{rot} \left(\cdot|0\right), p_{t}^{tor} \left(\cdot|0\right)`$'   <->   $`\mathbb{E_{x_tilde \sim \mathcal{N}(x,\,\sigma^{2}I)}}`$ 
    4) Algorithm4 of your paper   <->   Algorithm1 of Yang Song's, where T is set to be 1 and L to be N.

**Comment by YoungjunLee13** [comment]
For the 2nd question, it is almost resolved now. I had forgotten the presence of 'tr_sigma', 'rot_sigma_norm', and 'tor_score_norm2' in 'training.py'. Could you instead let me know how the reciprocal of 'rot_sigma_norm' serves the same purpose as 'rot_sigma'?
